### PR TITLE
relnote(117): Network Monitor shows proxy request info in Headers panel

### PR DESCRIPTION
**Pull request by bsmth**
Proxied requests now show proxy address, proxy status and proxy HTTP version in the Headers panel of the Network Monitor

__Related issues and pull requests:__
- [ ] Parent issue https://github.com/mdn/content/issues/28755

__Bugzilla:__
- https://bugzilla.mozilla.org/show_bug.cgi?id=1707192

### CODE REVIEW

**Comment by bsmth** [comment]
Copying screenshot from Bugzilla for illustration:

<details>

image

</details>

File: files/en-us/mozilla/firefox/releases/117/index.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ This article provides information about the changes in Firefox 117 that affect d
 
 ### Developer Tools
 
+- The Network Monitor now shows information about proxied requests, including the proxy address, proxy status and proxy HTTP version in the Headers panel ([Firefox bug 1707192](https://bugzil.la/1707192)).
```
Comment (dipikabh) [inline files/en-us/mozilla/firefox/releases/117/index.md:15]:
```suggestion
### Developer tools

- The Network Monitor now shows information about proxied requests, including the proxy address, the proxy status, and the proxy HTTP version in the Headers tab ([Firefox bug 1707192](https://bugzil.la/1707192)).
```

- Should we add this link on "Headers" tab: https://firefox-source-docs.mozilla.org/devtools-user/network_monitor/request_details/index.html?
- Also, I have the same comment about position of "Developer tools".

**Review by dipikabh — APPROVED** [review]
This too can land after you've considered the suggestions. Leaving a +1.

Thank you! 👍

**Comment by bsmth** [comment]
Thanks @dipikabh - I've applied your suggestions and merging shortly 👍🏻

**Pull request merged**